paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -996,6 +1101,17 @@ class PaymentSheet internal constructor(
                 ).create()
             }
 
+            /**
+             * Create the [FlowController] when launching [PaymentSheet] from a [ComponentActivity]
+             * and intending to create the [PaymentIntent] or [SetupIntent] on your server.
```

Comment:
Hrmm.  All integration variants create the Intent on the server, so we should think of something else.  

Maybe like this?
```suggestion
             * and you call the `presentWithIntentConfiguration` method`.

Reply:
Should we reword it entirely?

The normal flow:

```
Create a FlowController that you configure with the client secret of a PaymentIntent or SetupIntent.
```

The decoupled flow with client-side confirmation:

```
Create a FlowController that you configure with an IntentConfiguration. The resulting PaymentIntent or SetupIntent is created on your server, but confirmed on the client.
```

The decoupled flow with server-side confirmation:

```
Create a FlowController that you configure with an IntentConfiguration. The resulting PaymentIntent or SetupIntent is created and confirmed on your server.
```

(Because on second thought, the `when launching [PaymentSheet] from a [ComponentActivity]` part is redundant.)

Reply:
lg2m. I'll throw in some alternatives in case you like them more or gives you ideas

> "Create a FlowController that you configure with an IntentConfiguration. The PaymentIntent or SetupIntent is created on your server and confirmed on the client."

> "Create a FlowController that you configure with an IntentConfiguration by calling [configureWithIntentConfiguration]. Use this method if you confirm the PaymentIntent or SetupIntent on the client."

Reply:
I'd probably vote for over-explaining.